synapse/models/infotech.py

```diff
@@ -1171,12 +1172,12 @@ def getModelDefs(self):
                         'disp': {'hint': 'text'},
                     }),
                     ('url', ('inet:url', {}), {
-                        'doc': 'The URL that documents the ATT&CK tactic.',
-                    }),
+                        'doc': 'The URL that documents the ATT&CK tactic.'}),
+
                     ('tag', ('syn:tag', {}), {
-                        'doc': 'The synapse tag used to annotate nodes included in this ATT&CK tactic.',
-                        'ex': 'cno.mitre.ta0100',
-                    }),
+                        'deprecated': True,
+                        'doc': 'Deprecated. Please use ou:technique:tag.'}),
```

Comment:
Tactics are that squishy sort of "lifecycle phase" thing. I don't think redirecting them to ou:technique is correct here. This may be better pointed to the (not yet released) plan:* stuff that was supposed to account for ATT&CK Flow and such? Been a while since we had that talk...

Reply:
I'm not totally sure that i believe that the mitre tactic concept actually "exists" so i'd be fine with just saying `Deprecated.` and moving on. Does that work?

Reply:
:+1: